scripts/statistics.js

```diff
@@ -30,7 +41,10 @@ const processData = (data) => {
           stats.total.null++;
           real_value = false;
         }
-        if (checkSupport(data.support[browser], true)) {
+        if (
+          checkSupport(data.support[browser], true) ||
+          checkSupport(data.support[browser], '<=')
```

Comment:
Hm, we don't want to count this like it is a `true` value. 
I believe for the year goal "≤37" would be good enough, so this should probably be rather counted as real values or we could count it in an own category.

Also "<=" should be  "≤".